### PR TITLE
XRDDEV-525 XRDDEV-978 RHEL 7 and 8 packages

### PR DESCRIPTION
**Pull request by jansu76**
https://jira.niis.org/browse/XRDDEV-525
https://jira.niis.org/browse/XRDDEV-978

also remove some unused files such as nginx.p12

### CODE REVIEW

File: src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Name:               xroad-proxy-ui-api
+Version:            %{xroad_version}
+# release tag, e.g. 0.201508070816.el7 for snapshots and 1.el7 (for final releases)
+Release:            %{rel}%{?snapshot}%{?dist}
+Summary:            X-Road proxy UI REST API
+Group:              Applications/Internet
+License:            MIT
+BuildRequires:      systemd
+Requires(post):     systemd
+Requires(post):     /usr/sbin/semanage, /usr/sbin/setsebool
```
Comment (jhyoty) [inline src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec:15]: These seem to be unnecessary (can not find any SELinux rule changes)

File: src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+Requires(preun):    systemd
+Requires(postun):   systemd
+Requires:           net-tools, tar
```
Comment (jhyoty) [inline src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec:18]: xroad-proxy-ui-api-setup.sh needs `ip` and `hostname` (from _iproute_ and _hostname_ packages). net-tools should be unnecessary. Can't find any reference to `tar` either.

File: ansible/roles/xroad-base/tasks/rhel.yml [hunk 3]
```diff
@@ -42,10 +42,11 @@
...
-    group: root
-    mode: 0644
+# repo already added by step Setup the correct repository for X-Road installation (RHEL)
+#- name: xroad repo
+#  template:
+#    src: "xroad.repo.j2"
+#    dest: "/etc/yum.repos.d/xroad.repo"
+#    owner: root
+#    group: root
+#    mode: 0644
```
[the remark this answers is not in the document]
Comment (jansu76) [reply ansible/roles/xroad-base/tasks/rhel.yml]: fixed and created https://jira.niis.org/browse/XRDDEV-1200

File: src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec (hunk 1 above)
Comment (jansu76) [reply src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec]: fixed

File: src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec (hunk 2 above)
Comment (jansu76) [reply src/packages/src/xroad/redhat/SPECS/xroad-proxy-ui-api.spec]: fixed

**Pull request merged**